vita3k/renderer/src/vulkan/scene.cpp

```diff
@@ -235,10 +247,21 @@ static void bind_vertex_streams(VKContext &context, MemState &mem) {
 
     for (std::size_t i = 0; i < max_stream_idx; i++) {
         if (state.vertex_streams[i].data) {
+#ifdef __APPLE__
+            const bool restride = vertex_program.streams[i].stride % 4 != 0;
```

Comment:
Could you add a comment saying this is needed because MoltenVK forces the vertex stride to be a multiple of 4.